utils/compose/cmd.go

```diff
@@ -106,6 +107,41 @@ func (c *Command) Stop(args ...string) error {
 	return c.quietRun(c.cmd("stop", args))
 }
 
+func (c *Command) ExitedSuccessfully(services []string) error {
```

Comment:
It would be more useful to export serviceExitCode and add plain functions to do assertions about it.

Reply:
compose.Command now exports ServiceExitCode.